dataprep-pipelines/tests/integration.rs

```diff
@@ -1,44 +1,80 @@
+#[macro_use]
+extern crate lazy_static;
+extern crate rand;
+
 use dir_assert::assert_paths;
+use rand::Rng;
 use std::path::PathBuf;
 use tokio::fs;
+use tokio::io::AsyncWriteExt;
 
-#[tokio::test]
-async fn it_works_for_one_file() {
+lazy_static! {
+    static ref MANIFEST_FILE: PathBuf = PathBuf::from("test/manifest.json");
```

Comment:
nice!! did you get how this uses types in a weird way to make this work? not necessary to use it correctly, but it's a really cool hack